data/campaigns/The_South_Guard/utils/savescumming.cfg

```diff
@@ -0,0 +1,146 @@
+    {SET_GLOBAL_VARIABLE {VARNAME} $global_tmp}
+#enddef
+
+#-------------------
+# TRACK GAMESTATE
+#-------------------
+# every time something changes, increment both a global and local variable
+# after a load, compare the global and local variable - if they're different, we save-loaded
+
+# put this in the first scenario
+#define RESET_SAVELOAD_TIP
+    [event]
+        name=prestart
+        {SET_GLOBAL_VARIABLE sg_saveload_info "no"} # for some reason neither CLEAR_ nor setting to "" worked, but "no" does.
+    [/event]
+#enddef
+
+# put this in every scenario
+#define TRACK_SAVELOADS
+    [event]
+        name=prestart,side 1 turn end
+        first_time_only=no
+        {SET_GLOBAL_VARIABLE gamestate_global 0}
+        {           VARIABLE gamestate_local  0}
+        {SET_GLOBAL_VARIABLE saveloads_this_turn 0}
+    [/event]
+    [event]
+        name=new turn,moveto,sighted,recruit,recall,attacker hits,attacker misses,defender hits,defender misses,last breath,die
+        first_time_only=no
+
+        {GLOBAL_VARIABLE_OP gamestate_global add 1}
+        {       VARIABLE_OP gamestate_local  add 1}
+        [allow_undo]
+        [/allow_undo]
+    [/event]
+
+    [event]
+        name=preload
+        first_time_only=no
+        {FILTER_CONDITION({VARIABLE_CONDITIONAL enable_tutorial_elements equals yes})}
+        {GET_GLOBAL_VARIABLE gamestate_global}
+        [if] {VARIABLE_CONDITIONAL global_tmp not_equals $gamestate_local}
+            [then]
+                {GLOBAL_VARIABLE_OP saveloads_this_turn add 1}
+            [/then]
+        [/if]
+        {CLEAR_VARIABLE global_tmp}
+        # if we've save-loaded 4 times this turn
+        {GET_GLOBAL_VARIABLE saveloads_this_turn}
+        [if] {VARIABLE_CONDITIONAL global_tmp greater_than_equal_to 4}
+            [then]
+                # and we haven't already shown this tip
+                {GET_GLOBAL_VARIABLE sg_saveload_info}
+                [if] {VARIABLE_CONDITIONAL global_tmp not_equals yes}
+                    [then]
+                        {SET_GLOBAL_VARIABLE sg_saveload_info yes}
+                        [display_tip]
+                            title=_"Save-Loading and RNG"
+                            image=tutor/saveload.png
+                            message=_"I notice you’ve been loading a lot
+of saved games. If this is how you prefer to play, 
+more power to you!
+
+But if you’re save-loading because hits and missess
+feel unfair, try restarting the campaign and 
+selecting <i><b>“Reduced RNG”</b></i> instead of <i><b>“Default RNG”</b></i>.
+You’re also welcome to turn down the difficulty:
+when you load a “start-of-scenario” save, there’s 
+a little <i><b>“change difficulty”</b></i> box you can check.
+
+Alternatively, you may want to go back to a previous
+turn or even scenario, to choose a different strategy
+or carry over additional gold.
+
+We balance campaigns around minimal save-loading 
+and have ironmanned all campaigns during development.
```

Comment:
What does 'ironman' mean in this context? I'm not familiar with this word/term as a verb.

Reply:
Nothing so brief. If you're not concerned with brevity, I would amend to 'We balance campaigns around minimal save-loading and have play-tested all campaigns during development to verify that save-loading is not necessary.' Or something to that effect.